sechub-pds-solutions/prepare/README.adoc

```diff
@@ -37,7 +37,11 @@ The folder contains a start script which does the manual steps for you:
 
 The container will be started and attached to the `sechub` network.
 
-WARNING: Make sure the SecHub container is running and executor for Prepare is set up.
+WARNING: Make sure the SecHub container is running with compose prepare and executor for Prepare is set up.
+
+. Adjust the docker-compose_pds_prepare_external-network.yml file and set the volume for the shared_volume directory to your local .sechub folder +
```

Comment:
This should not be necessary - please look at my comments below (I already mentioned this).

In a nutshell:
- user may not need to change something
- tmp/sechub-shared-volume could be an option for this
- shall always work out-of-the-box